lib/helpers/answerHelper.js

```diff
@@ -116,12 +116,66 @@ function summaryFormatter(answerObject, fullUiSchema, answerLookup) {
     return answerIndex;
 }
 
+function objectFormatter(obj) {
+    const isArray = Array.isArray(obj);
+    let returnValue = '';
+    if (isArray) {
+        obj.forEach(answer => {
+            // eslint-disable-next-line no-use-before-define
+            const formattedAnswer = formatAnswerV2(answer);
+            returnValue += `${formattedAnswer}<br>`;
+        });
+        return returnValue.substring(0, returnValue.length - 4);
+    }
+    let lineEnd = '';
+    Object.keys(obj).forEach(answer => {
+        lineEnd = Object.keys(obj).length > 3 ? '<br>' : ' '; // For large numbers of fields, insert a line break.
+        if (obj[answer] !== '') {
+            // Ignore empty lines
+            // eslint-disable-next-line no-use-before-define
+            const formattedAnswer = formatAnswerV2(obj[answer]); // Format the individual answer.
+            lineEnd = formattedAnswer === 'Yes' || formattedAnswer === 'No' ? ', ' : lineEnd; // If a question was a boolean, follow it with a comma, not a space.
+            returnValue += `${formattedAnswer}${lineEnd}`; // append the answer and the line end to the return string.
+        }
+    });
+    return returnValue.substring(0, returnValue.length - lineEnd.length);
+}
+
+function dateFormatterV2(date, format) {
+    return moment(date).format(format);
+}
+
+function parseDateFormat(datePrecision) {
+    if (/^[yY\-/\s]*$/.test(datePrecision)) {
+        return 'YYYY';
+    }
+    if (/^[ymYM\-/\s]*$/.test(datePrecision)) {
+        return 'MMMM YYYY';
+    }
+    return 'DD MMMM YYYY';
+}
+
+function formatAnswerV2(answer, format) {
+    if (format && format.value && format.value === 'date-time') {
+        const dateFormat = parseDateFormat(format.precision);
+        return dateFormatterV2(answer, dateFormat);
+    }
+    if (typeof answer === 'object' && answer !== null) {
```

Comment:
is `typeof answer === 'object'` redundant given the `Array.isArray(obj);` check in the `objectFormatter` function?

Reply:
No, It's possible for this function to be called on a string, hence the fall thru.